picard/script/functions.py

```diff
@@ -1354,3 +1355,16 @@ def func_unique(parser, multi, case_sensitive="", separator=MULTI_VALUED_JOINER)
     if not case_sensitive:
         multi_value._multi = list({v.lower(): v for v in multi_value}.values())
     return multi_value.separator.join(sorted(set(multi_value)))
+
+
+@script_function(documentation=N_(
+    """`$countryname(country_code)`
+
+Returns the name of the country for the specified country code.  If the country code is invalid an empty string will be returned.
+"""
+))
+def func_countryname(parser, country_code):
+    code = country_code.strip().upper()
+    if code not in RELEASE_COUNTRIES:
+        return ''
+    return RELEASE_COUNTRIES[code]
```

Comment:
```suggestion
    return RELEASE_COUNTRIES.get(code, "")
```

Reply:
How about:
```
    return _(RELEASE_COUNTRIES.get(code, ""))
```
to allow for translation into local language.

This would require `picard.const.countries.py` to be changed so that each country was e.g.
```
    'AE': N_('United Arab Emirates'),
```
You might want a parameter e.g. `$countryname(code,translate)` where translate is non-blank to enable this translation.

Reply:
> This would require `picard.const.countries.py` to be changed so that each country was e.g.

Actually the translations are already available through the `picard.i18n` module.

> You might want a parameter e.g. `$countryname(code,translate)` where translate is non-blank to enable this translation.

Good idea.  Optional parameter added.